File: DemoApp/DemoApp/NativeBannerView.m
//
//  NativeBannerView.m
//  DemoApp
//
//  Created by Glispa GmbH on 12/29/15.
//  Copyright © 2015 Glispa GmbH All rights reserved.
//

#import "NativeBannerView.h"


@implementation NativeBannerView

+ (CGFloat)desiredHeight {
    return 60;
}


#pragma mark - Overriden methods


// Implement + (NSString*)xibName only if this class is related with XIB
+ (NSString *)xibName {
    return NSStringFromClass([self class]);
}


@end

File: DemoApp/DemoAppSwift/NativeBannerView.swift
//
//  NativeBannerView.swift
//  DemoApp
//
//  Created by Glispa GmbH on 19/05/16.
//  Copyright © 2016 Glispa GmbH. All rights reserved.
//

import UIKit
import AvocarrotNativeView

class NativeBannerView: AVONativeView {

    static func desiredHeight() -> CGFloat {
        return 60.0
    }

    override static func xibName() -> String {
        return NSStringFromClass(self)
    }

}
